Regression/WarpX-tests.ini

```diff
@@ -2034,7 +2034,6 @@ numthreads = 1
 compileTest = 0
 doVis = 0
 compareParticles = 0
-analysisRoutine = Examples/Modules/embedded_boundary_cube/analysis_fields.py
```

Comment:
@RemiLehe When I merged in the latest development branch I had a conflict in this line. Is it correct that this line shouldn't be included? The file `Examples/Modules/embedded_boundary_cube/analysis_fields.py` does exist so I'm just trying to make sure the analysis for the test wasn't accidentally dropped.